compositor_api/src/types/component.rs

```diff
@@ -92,6 +92,18 @@ pub struct View {
 
     /// List of box shadows.
     pub box_shadow: Option<Vec<BoxShadow>>,
+
+    /// (**default=`0.0`**) Padding on top side in pixels.
+    pub padding_top: Option<f32>,
```

Comment:
I think we talked about adding also option padding and maybe (padding_vertical and horizontal)

Reply:
It was added only to react sdk but I'll add it to api as well